runtime/floats.c

```diff
@@ -93,8 +93,8 @@ CAMLexport void caml_Store_double_val(value val, double dbl)
 
   CAMLassert(sizeof(double) == 2 * sizeof(value));
   buffer.d = dbl;
-  Op_val(val)[0] = buffer.v[0];
-  Op_val(val)[1] = buffer.v[1];
+  Field(val, 0) = buffer.v[0];
+  Field(val, 0) = buffer.v[1];
```

Comment:
This looks like a typo, shouldn't it be:
```C
Field(val, 1) = buffer.v[1];
```

Reply:
They were typos indeed, well spotted, thanks!